frontend/src/App/App.tsx

```diff
@@ -88,6 +84,15 @@ function App() {
     setMurals(data.rows);
   };
 
+  /**
+   * Zooms the map to the coordiantes of the clicked searched mural
+   * @param long longitude
+   * @param lat latitude
+   */
+  const handleSearchedMuralZoom = (long: number, lat: number) => {
+    mapRef.current.setLongLat(long, lat);
```

Comment:
another option is to use react-map-gl's FlyToInterpolator method ([docs](https://github.com/visgl/react-map-gl/blob/master/docs/advanced/viewport-transition.md#example-fly-to-a-new-location))

Reply:
Would it also make sense to close the search menu once you click a mural?

Reply:
yep 100%